src/collector.ts

```diff
@@ -105,13 +105,38 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
+
+        function addReplaceDependency(line: string, index: number) {
+            // split the replace statements by '=>'
+            const parts: Array<string> = line.replace('replace', '').replace('(', '').replace(')', '').trim().split('=>');
+            regExp.lastIndex = 0;
```

Comment:
This looks odd, create a function which returns new regExp instance on each need. Also the name `regExp` is too generic. semVerRegExp?